discord/app/context.py

```diff
@@ -29,9 +29,10 @@
 
 from ..interactions import Interaction
 from ..utils import cached_property
+from ..abc import Messageable
 
 
-class InteractionContext:
+class InteractionContext(Messageable):
```

Comment:
Not related to this PR.

Reply:
I had to do that so I could check for a messageable. The original context is considered a messageable, just made sense to me that this should be too

Reply:
That's fine, but let's do it in another PR.

Reply:
I strongly prefer we don't merge unrelated changes like this on PRs. It promotes sloppiness when we have an undefined scope, and is how bugs slip in.